requirements.txt

```diff
@@ -6,4 +6,4 @@ matrix_client>=0.3.2,<4.0
 pycryptodome>=3.6.6,<4.0
 web3>=4.5.0,<5.0
 
-git+https://github.com/raiden-network/raiden-contracts.git@d77e2a7f1f83afd302caabf1179374879bc7725c#egg=raiden-contracts
+raiden-contracts==0.1.6
```

Comment:
man, this is the raiden-contracts version. It should be `0.4.0`

Reply:
aaarrgh. I need to get some caffeine now